src/main/java/com/jfrog/ide/idea/scan/ScanManager.java

```diff
@@ -150,10 +150,6 @@ public void run(@NotNull com.intellij.openapi.progress.ProgressIndicator indicat
                 if (project.isDisposed()) {
                     return;
                 }
-                if (!GlobalSettings.getInstance().areXrayCredentialsSet()) {
-                    getLog().warn("Xray server is not configured.");
-                    return;
-                }
```

Comment:
What is the reason for removing this?

Reply:
Because as much as I saw it was redundant.
https://github.com/jfrog/jfrog-idea-plugin/blob/master/src/main/java/com/jfrog/ide/idea/scan/ScanManagersFactory.java#L87-L90
What do you think?

Reply:
Scans can be triggered also by a change in `package-json.lock`, `go.mod`, `pom.xml`, and more. Scans triggered by a change in the project descriptor do not need the scan managers' factory.
We would prefer to avoid triggering scans when the credentials are not configured, since it performs heavy operations (like building the dependency tree) for nothing.

Reply:
I'm not sure it is a valid scenario after we added the new JFrog CLI config logic. Anyway, are you fine with the original double-check?